content/Video_Display.md

```diff
@@ -553,27 +553,28 @@ the one at `0:9800`!
 ### Background (BG)
 
 The [SCY and SCX](#ff42-scy-scroll-y-r-w-ff43-scx-scroll-x-r-w) registers can be
-used to scroll the background, allowing to select the origin of the visible
-160x144 pixel area within the total 256x256 pixel background map.
+used to scroll the Background, specifying the origin of the visible
+160x144 pixel area within the total 256x256 pixel Background map.
 The Background visible area wraps around the Background map (that is, when part of
 the visible area goes beyond the map edge, it starts displaying the opposite side of the map).
 
-Whether the background is displayed can be toggled using
-[LCDC bit 0](#lcdc-0-bg-window-display-priority), except on CGB in CGB Mode,
-where it's always drawn.
+In Non-CGB mode, the Background (and the Window) can be disabled using
```

Comment:
Maybe "Outside of CGB in CGB mode", since it also applies to DMG? To be consistent with the LCDC.0 section title...

Reply:
Actually, I think that it is the LCDC.0 title what can be improved. I have changed it a bit.